core/services/ardupilot_manager/firmware_download/FirmwareDownload.py

```diff
@@ -38,8 +39,7 @@ class Platform(str, Enum):
 
     Pixhawk1 = "Pixhawk1"
     Navigator = "Navigator"
-    SITL_ARM = "SITL_arm_linux_gnueabihf"
-    SITL_X86 = "SITL_x86_64_linux_gnu"
+    SITL = "SITL"
```

Comment:
That is wrong, you should use a function that generates the string, check my first comment

Reply:
Ohhhh ok, now I get it. This seems more reasonable. Will change.

Reply:
What do you think about this last update?